presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotClusterInfoFetcher.java

```diff
@@ -193,24 +193,27 @@ public String doHttpActionWithHeaders(
 
     private String sendHttpGetToController(String path)
     {
+        final String controllerPath = format("%s://%s/%s", pinotConfig.isUseHttpsForController() ? "https" : "http", getControllerUrl(), path);
```

Comment:
Can we use `HttpUriBuilder` from airlift?

Reply:
sounds good